dts/arm/seeed/lora-e5.dtsi

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2021 Thomas Stranger
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <st/wl/stm32wle5Xc.dtsi>
+#include <st/wl/stm32wle5jcix-pinctrl.dtsi>
+
+&clk_hse {
+	hse-bypass;
```

Comment:
Missing `clock-frequency `property ?

Reply:
stm32wl.dtsi already defines the correct frequency for this series:
```
/* Expected clock-frequency on the whole series 32MHz */
clock-frequency = <DT_FREQ_M(32)>;
```
I thought about re-defining it anyways and decided initially against it.
Will add it now anyways.